docs/apache-airflow/concepts/operators.rst

```diff
@@ -66,20 +66,20 @@ Jinja Templating
 ----------------
 Airflow leverages the power of `Jinja Templating <http://jinja.pocoo.org/docs/dev/>`_ and this can be a powerful tool to use in combination with :ref:`macros <templates-ref>`.
 
-For example, say you want to pass the execution date as an environment variable to a Bash script using the ``BashOperator``:
+For example, say you want to pass the start of the data interval as an environment variable to a Bash script using the ``BashOperator``:
 
 .. code-block:: python
 
-  # The execution date as YYYY-MM-DD
+  # The start of the data interval as YYYY-MM-DD
   date = "{{ ds }}"
```

Comment:
Should we use something other than `ds` here now?

Reply:
I guess we can use `{{ run_id }}`.

Reply:
I was wondering if we are deprecating `{{ ds }}` -- i we aren't then this can stay as it is.

Reply:
We are not, but the semantic meaning changed (it is now backed by `data_interval_start`, which is another reason why logical date is hard wired to equal to `data_interval_start` right now).